test/Microsoft.TestPlatform.TestUtilities/IntegrationTestBase.cs

```diff
@@ -65,6 +69,20 @@ public IntegrationTestBase()
     public string StdErr => _standardTestError;
     public string StdErrWithWhiteSpace { get; private set; } = string.Empty;
 
+    public TempDirectory TempDirectory { get; }
+
+    public TestContext TestContext { get; set; }
+
+    [TestCleanup]
+    public void TempDirectoryCleanup()
+    {
+        // Delete the directory only when the test succeeded, so we can look at results and logs of failed tests.
```

Comment:
Maybe path to the directory is already displayed. If not, could we (I don't remember how to do so) add it to the output?

Reply:
it is in the output.